GiveNGo/GiveNGo
# Add multiple tasks and requests to home page

## Description:
- This change adds multiple tasks and requests to the Home Page. Each Request and Task displays the description and an icon based on the category of the task. Types file was also added. Initial setup was added to redirect to TaskDetail page but this functionality is not complete yet. 

## Changes I Made:
- [X] In Footer.tsx, update Home tab to navigate to HomeRoute
- [X] Create HomeRouter.tsx. This file is used to navigate between the home page and task detail page. 
- [X] In RequestCard.tsx, create button for each request along with icon and description.
- [X] In TaskCard.tsx, create button for each request along with icon and description.
- [X] Create TaskDetail.tsx. This file will be used to display detailed information about a task.
- [X] In HomePage.tsx, create dummy data and create multiple tasks and requests with props for individual tasks. 
- [X] Create types.tsx to store types information for TaskCard and ProjectCard.

## How to Test:
1. npm install
2. expo start
3. Open the application on your phone or your emulator
4. Login and click accept.
5. This will navigate to the Home Page. This should look like the screenshot below.

![HomePR13](https://user-images.githubusercontent.com/7851187/88259201-ad10fb80-cc76-11ea-93d3-e8ea0d227ea4.png)
